vmware-base/src/main/java/com/cloud/hypervisor/vmware/mo/DatacenterMO.java

```diff
@@ -65,24 +71,7 @@ public DatacenterMO(VmwareContext context, String dcName) throws Exception {
 
     @Override
     public String getName() throws Exception {
-        return (String)_context.getVimClient().getDynamicProperty(_mor, "name");
-    }
-
-    public void registerTemplate(ManagedObjectReference morHost, String datastoreName, String templateName, String templateFileName) throws Exception {
```

Comment:
Were these removed methods unused?

Reply:
yes, planning to go through the whole vmware-base lib and remove even more.